Makefile

```diff
@@ -448,7 +448,7 @@ endif
 	echo "base/version_git.jl" > light-source-dist.tmp
 
 	# Download all stdlibs and include the tarball filenames in light-source-dist.tmp
-	@$(MAKE) -C stdlib getall NO_GIT=1
+	@$(MAKE) -C stdlib getall DEPS_GIT=0 USE_BINARYBUILDER=0
```

Comment:
`light-source` is not supposed to contain any dependency source; only Julia's source.  Specifying `USE_BINARYBUILDER=0` doesn't really make sense here.

Reply:
Agreed, this was just future-proofing against it deciding to do something foolish later